### PR TITLE
Tracking for bot installation and linking fyle account

### PR DESCRIPTION
**Pull request by shreyanshss7**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,9 +1,11 @@
 aiohttp==3.8.1
 aiosignal==1.2.0
+analytics-python==1.4.0
 asgiref==3.4.1
 astroid==2.9.0
 async-timeout==4.0.1
 attrs==21.2.0
+backoff==1.10.0
```
Comment (gokulk16) [inline requirements.txt:8]: not seeing this been used in this PR.
If need to be used later, can u add to that PR

File: requirements.txt [hunk 2]
```diff
@@ -26,6 +28,7 @@ isort==5.10.1
 jsonpickle==1.4.2
 lazy-object-proxy==1.6.0
 mccabe==0.6.1
+monotonic==1.6
```
Comment (gokulk16) [inline requirements.txt:31]: remove if not used in this PR

File: requirements.txt [hunk 3]
```diff
@@ -35,6 +38,7 @@ psycopg2-binary==2.9.2
 pycparser==2.21
 PyJWT==1.7.1
 pylint==2.12.1
+python-dateutil==2.8.2
```
Comment (gokulk16) [inline requirements.txt:41]: remove if not used in thisPR

File: requirements.txt (hunk 1 above)
Comment (shreyanshss7) [reply requirements.txt:8]: These are dependent libraries of the `analytics-python` most probably, some libraries get auto installed if they are used internally in a library.

File: requirements.txt (hunk 3 above)
Comment (shreyanshss7) [reply requirements.txt:41]: Same as above

File: requirements.txt (hunk 2 above)
Comment (shreyanshss7) [reply requirements.txt:31]: Same as above

**Review by abhishek1234321 — APPROVED** [review]

**Pull request merged**